modules/nw-sriov-configuring-operator.adoc

```diff
@@ -6,21 +6,35 @@
 [id="nw-sriov-configuring-operator_{context}"]
 = Configuring the SR-IOV Network Operator
 
-[IMPORTANT]
+. Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator related components.
+
+.. Create a file named `sriovOperatorConfig.yaml` using the following YAML:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default 
+  namespace: openshift-sriov-network-operator 
+spec:
+  disableDrain: false
+  enableInjector: true
+  enableOperatorWebhook: true
+  logLevel: 2
+----
++
+[NOTE]
 ====
-Modifying the SR-IOV Network Operator configuration is not normally necessary.
-The default configuration is recommended for most use cases.
-Complete the steps to modify the relevant configuration only if the default behavior of the Operator is not compatible with your use case.
+The only valid name for the `sriovOperatorConfig` is `default` and it must be on the namespace where the operator is deployed. 
 ====
 
-The SR-IOV Network Operator adds the `SriovOperatorConfig.sriovnetwork.openshift.io` CustomResourceDefinition resource.
-The Operator automatically creates a SriovOperatorConfig custom resource (CR) named `default` in the `openshift-sriov-network-operator` namespace.
-
-[NOTE]
-=====
-The `default` CR contains the SR-IOV Network Operator configuration for your cluster.
-To change the Operator configuration, you must modify this CR.
-=====
+.. Create the object by running the following command:
```

Comment:
I suggested adding "resource" above, but "object" would be fine too. But whichever you go with, I'd use the same for both